### ISSUE TITLE
A problem in augmentation of training data

### ISSUE THREAD

**Issue by rose-jinyang** [jac99/FootAndBall#10]
Hello
How are you?
Thanks for contributing to this project.
I found that you used a random crop of images in the training step.
If we random-crop the images, I think that the boxes info should be changed too.
But I did not find any process for boxes in your project?

**Comment by jac99** [comment]
It's not easy to deciced when and how bounding boxes should be modified when random crop is applied. When only part of the player is erased, the bounding box should be unchanged I think. When the entire player is erased, we should remove the bounding box.
But I haven't implemented such functionally - the ground truth bounding boxes are never removed. 
You can try to modify the data augmentation procedures to remove ground truth bounding boxes when an entire player is masked. But I don't think it'll have a significant impact on the performance.

**Comment by rose-jinyang** [comment]
Thanks for your reply.

**Comment by rose-jinyang** [comment]
Hi @jac99 
How are you?
I have a question.
You used the BalancedSampler for data loader.
What would happen if we do not use the customized BalancedSampler?
In fact, I am training a mode without using this BalancedSampler on my custom dataset but the model does not converge well.
